app/hooks/handle_send_message.test.tsx

```diff
@@ -69,6 +74,9 @@ describe('useHandleSendMessage', () => {
 
     it('should handle basic message send', async () => {
         const {result} = renderHook(() => useHandleSendMessage(defaultProps), {wrapper});
+        jest.mocked(createPost).mockResolvedValueOnce({
+            data: true,
+        });
```

Comment:
You can add this to the `beforeEach`, or even just after `jest.mock(...)`. That way you don't have to write it on every one, you just have to overwrite it in the one you need.

If you add it at the beginning of the file, remember to use `mockResolvedValue` instead of `mockResolvedValueOnce`.

Reply:
Sure.